src/main/java/com/cdancy/bitbucket/rest/features/BranchApi.java

```diff
@@ -114,4 +119,35 @@ Branch getDefault(@PathParam("project") String project,
     @GET
     BranchModel model(@PathParam("project") String project,
                       @PathParam("repo") String repo);
+
+    @Named("branch:get-BranchPermission")
+    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/4.14.1/bitbucket-ref-restriction-rest.html#idm45354011023456"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/branch-permissions/2.0/projects/{project}/repos/{repo}/restrictions")
+    @Fallback(BitbucketFallbacks.BranchPermissionPageOnError.class)
+    @GET
+    BranchPermissionPage getBranchPermission(@PathParam("project") String project,
```

Comment:
Lets change to `listBranchPermission` to keep in line with naming standards already set.